pkg/operator/ceph/object/config_test.go

```diff
@@ -178,7 +177,7 @@ func Test_clusterConfig_generateMonConfigOptions(t *testing.T) {
 				t.Errorf("clusterConfig.generateMonConfigOptions() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			assert.Equal(t, fmt.Sprint(tt.want), fmt.Sprint(got)) // go maps are stringified in sorted order for testing
+			assert.Equal(t, tt.want, got)
```

Comment:
Underneath, `assert.Equal()` uses `assert.ObjectsAreEqual()` which will use `reflect.DeepEqual()` to test maps against each other.